src/test/java/com/williamfiset/algorithms/graphtheory/BridgesAdjacencyListIterativeTest.java

```diff
@@ -81,7 +81,6 @@ public void graphWithCyclesTest() {
     addEdge(graph, 11, 6);
 
     BridgesAdjacencyList solver = new BridgesAdjacencyList(graph, n);
-    List<Integer> bridges = solver.findBridges();
     List<Pair<Integer, Integer>> sortedBridges = getSortedBridges(solver.findBridges());
```

Comment:
Optional: pass in bridges to getSortedBridges?

Reply:
that's exactly what it does :-)
List<Pair<Integer, Integer>> sortedBridges = getSortedBridges(bridges);
would be just the same,